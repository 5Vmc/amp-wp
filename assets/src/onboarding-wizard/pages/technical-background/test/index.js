
/**
 * External dependencies
 */
import { create } from 'react-test-renderer';

/**
 * Internal dependencies
 */
import { TechnicalBackground } from '..';
import { Providers } from '../../..';

jest.mock( '../../../components/options-context-provider' );

describe( 'TechnicalBackground', () => {
	it( 'matches snapshot', () => {
		const wrapper = create(
			<Providers>
				<TechnicalBackground />
			</Providers>,
		);
		expect( wrapper.toJSON() ).toMatchSnapshot();
	} );
} );
