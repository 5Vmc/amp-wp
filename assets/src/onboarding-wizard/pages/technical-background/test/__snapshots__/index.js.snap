// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TechnicalBackground matches snapshot 1`] = `
<div
  className="error-screen-container"
>
  <div
    className="error-screen components-panel"
  >
    <h1>
      The setup wizard has experienced an error.
    </h1>
    <p>
      <span
        dangerouslySetInnerHTML={
          Object {
            "__html": "Cannot access 'error' before initialization",
          }
        }
      />
       
      <a
        href="http://site.test/wp-admin/?page=amp-options"
      >
        Return to AMP settings.
      </a>
    </p>
  </div>
</div>
`;
